iTaylorCodes/Capstone-2-Neighborhood-fe
# Add delete recent searches feature

### What does this PR do?
- This PR adds the feature to delete recent searches.
- This PR closes #36 
### Description of Task to be completed
- Add "Clear" button to Recent Searches section of homepage.
### How should this be manually tested?
- After performing searches and verifying they have been added to the Recent Searches Carousel, click "Clear" button and ensure all recent searches are deleted.

https://user-images.githubusercontent.com/76837421/149066538-0612b245-e3c8-46fb-9d1d-cf7bc5a90b22.mov
